src/coord_seq.rs

```diff
@@ -92,9 +92,7 @@ impl CoordSeq {
 
         if size > 0 {
             let dims = data[0].as_ref().len();
-            if let Err(e) = CoordDimensions::try_from(dims as u32) {
-                return Err(Error::GenericError(e.to_owned()));
-            }
+            let _ = CoordDimensions::try_from(dims as u32)?;
```

Comment:
I think:

```suggestion
            CoordDimensions::try_from(dims as u32)?;
```

works fine, no?

Reply:
Not sure clippy would be happy with unused function return values ?